worlds/minecraft/Options.py

```diff
@@ -14,15 +14,15 @@ class EggShardsRequired(Range):
     """Number of dragon egg shards to collect to spawn bosses."""
     display_name = "Egg Shards Required"
     range_start = 0
-    range_end = 74
+    range_end = 50
     default = 0
 
 
 class EggShardsAvailable(Range):
     """Number of dragon egg shards available to collect."""
     display_name = "Egg Shards Available"
     range_start = 0
-    range_end = 74
+    range_end = 50
```

Comment:
So @espeon65536 if I understand this correctly this is the lowest max possible if both bosses are turned on, exclude post-game, and all hard/unreasonable are turned off as well?

Basically the limit of free slots after using settings to exclude the maximum number of checks?

Reply:
That's correct. (It's actually like 52, but I think 50 is a nicer number + it prevents some possible problems where all items are progression.)